spec/models/video_spec.rb

```diff
@@ -1,15 +1,7 @@
 require 'spec_helper'
 
 describe Video do 
-  it "saves itself" do 
-    video = Video.new(title: "Archer", description: "Funniest show on tv.")
-    video.save 
-    expect(Video.first).to eq(video)
-  end
-
-  it "belongs to category" do 
-    cat = Category.create(name: "Comedies")
-    video = Video.create(title: "Archer", description: "Funny.", category: cat)
-    expect(video.category).to eq(cat)
-  end
+  it { should belong_to(:category)}
```

Comment:
Keep spacing consistent `it { should belong_to(:category)}` could be `it { should belong_to(:category) }`